run-tests.sh

```diff
@@ -27,5 +27,5 @@ pydocstyle invenio_migrator tests && \
 isort -rc -c -df **/*.py && \
 check-manifest --ignore ".travis-*" && \
 sphinx-build -qnNW docs docs/_build/html && \
-python setup.py test && \
+py.test tests/unit/ && \
```

Comment:
Have you consider using different pytest loaders based on installed invenio version?